hack/check/check-copyright.sh

```diff
@@ -1,88 +0,0 @@
-#!/bin/bash
```

Comment:
why remove the check-copyright file? I remembered it is required in stolostron.

Reply:
the copyright is not required in the stolostron

Reply:
I suggest to keep the copyright check. That is best practice to check it. If it does not work now, let us fix it in future.

Reply:
done